content/introduction/overview.md

```diff
@@ -47,36 +51,34 @@ What about that link to the Aardvark page above? It looked a little unusual:
-
-- Content can’t be moved to a different address. If a company reorganizes their web site today, they could move a page from `http://mycompany.com/what_we_do` to `http://mycompany.com/services`. In IPFS, the old link you have would still point to the same content.
+- If the text on a web page is changed, the new version gets a new, different link.
+- Content can’t be moved to a different address. On today's internet, a company could reorganize content on their website and move a page at  `http://mycompany.com/what_we_do` to `http://mycompany.com/services`. In IPFS, the old link you have would still point to the same content.
 
-- If someone removes a page entirely, you can still get it at the same address.
+Of course, people want to update and change content all the time, and don't want to send new links every time they do it. Check out the concept guides on [IPNS]({{<relref "guides/concepts/ipns.md">}}) and the [Mutable File System (MFS)]({{<relref "guides/concepts/mfs.md">}}) to learn more about how changing content can work in a content-addressed, distributed system.
 
-It’s important to remember in all of these situations that it’s the address or link that doesn’t change. The IPFS network is a participatory and collaborative one — if nobody has the content identified by a given address, it won’t be available. On the other hand, content can’t be removed from IPFS so long as *someone* is interested enough to make it available, whether or not that person is the original author.
+It’s important to remember in all of these situations using IPFS is participatory and collaborative. If nobody has the content identified by a given address available for others to access, you won't be able to get it. On the other hand, content can’t be removed from IPFS as long as *someone* is interested enough to make it available, whether that person is the original author or not.
 
 
 ## It’s all about possession and participation.
 
-While there is lots of complex technology in IPFS, the fundamental ideas are about changing how networks of people and computers communicate: while today’s world-wide-web is structured on *ownership* and *access*, where you get files from whoever owns them and they choose to grant you access, IPFS is based on the ideas of *possession* and *participation*, where many people have each others’ files and *participate* in making them available.
+While there's lots of complex technology in IPFS, the fundamental ideas are about changing how networks of people and computers communicate. Today’s world-wide-web is structured on *ownership* and *access*, where you get files from whoever owns them and they choose to grant you access. IPFS is based on the ideas of *possession* and *participation*, where many people have each others’ files and *participate* in making them available.
```

Comment:
"Today's World Wide Web is structured on *ownership* and *access*, meaning that you get files from whoever owns them -- if they choose to grand you access." (clarity, AP style)

Reply:
Done!